peepholelib/coreVectors/parse_ds.py

```diff
@@ -55,18 +56,21 @@ def parse_ds(self, **kwargs):
             if verbose: print(f'Allocating {key_list}')
 
             # dry run to get shapes
-            data = ds_parser([ds._dss[ds_key][0]])
+            
+            data = ds.get(ds_key, 0)
+
             with torch.no_grad():
-                _res = model(data['image'].to(device))
+                _res = model(data['image'].unsqueeze(0).to(device))
```

Comment:
We should not need to `unsqueeze` here. Probably this should be covered on `get()`